client/src/views/files/FoldersPage.vue

```diff
@@ -345,18 +340,19 @@ async function importFiles(): Promise<void> {
 }
 
 function selectAllFiles(checked: boolean): void {
-  for (const item of fileItemRefs.value || []) {
+  for (const item of fileListItemRefs.value || fileGridItemRefs.value) {
```

Comment:
Better to use `displayView` to choose the list to iterate on. I'm not exactly sure what happens with `refs` when they're shown/hidden.